Wikipedia/UI-V5/WMFArticlePreviewCell.m

```diff
@@ -24,6 +30,18 @@ - (void)prepareForReuse {
     self.summaryLabel.text = nil;
 }
 
+-(void)awakeFromNib {
+    [self rememberSettingsFromIB];
+}
+
+-(void)rememberSettingsFromIB {
+    NSRange r = NSMakeRange(0, 1);
+    self.summaryLabelAttributesFromIB = [self.summaryLabel.attributedText attributesAtIndex:0 effectiveRange:&r];
+    self.paddingAboveDescriptionFromIB = self.paddingConstraintAboveDescription.constant;
```

Comment:
you could also do all of it in masonry, and write visual tests to verify the appearance of each state instead of relying on IB... just sayin' :trollface: